CMakeLists.txt

```diff
@@ -105,6 +105,11 @@ INCLUDE(ThirdpartyToolchain)
 set (EXAMPLE_DIRECTORY ${CMAKE_SOURCE_DIR}/examples)
 
 add_subdirectory(c++)
+
+install(
+  FILES LICENSE NOTICE
+  DESTINATION .)
```

Comment:
If I do `make install`, these just get placed in my root directory. Other apache projects (e.g. arrow, parquet-cpp) don't install the `LICENSE` or `NOTICE` (the header files all have the license information), so why is this needed? This is why I removed these in #192, sorry if there was a good reason for installing these.